cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -186,10 +192,9 @@ image::rosa-ui-account-21.png[]
 +
 image::rosa-ui-billing-22.png[]
 +
-* Only AWS accounts that were linked to the Red{nbsp}Hat organization that is currently used will be shown.
-* The specified AWS account will be charged for using the ROSA service, regardless of whether the infrastructure AWS account is linked to it in the same AWS organization.
+* Only AWS accounts that were linked to the Red{nbsp}Hat organization that is currently used are shown.
+* The specified AWS account is charged for using the ROSA service, regardless of whether the infrastructure AWS account is linked to it in the same AWS organization.
 * You can see an indicator whether the ROSA contract is enabled for a given AWS billing account or not.
-* In case you would like to use an AWS account that does not have a contract enabled yet, you can either use the _Connect ROSA to a new AWS billing account_ to reach the ROSA AWS console page, where you can activate it after logging in using the respective AWS account by following steps described earlier in this tutorial, or ask the administrator of the AWS account to do that for you.
```

Comment:
I would add two more bullet points at the end of the list that further clarifies the "Contract enabled" label:
* AWS accounts without the _Contract enabled_ label are charged the applicable on-demand consumption rates.
* If you select an AWS billing account that shows the _Contract enabled_ label, on-demand consumption rates are charged only after the contracted quota is consumed.